src/pages/StockListing/StockListing.tsx

```diff
@@ -24,12 +24,32 @@ const StockListing = (): JSX.Element => {
   const howManyToDisplay: number = 10;
 
   useEffect(() => {
-    const fetchData = async () => {
+    // check for logged on User
+    const loggedUser = window.localStorage.getItem('loggedOnUser')
+    if(loggedUser){
+      const fetchUserData = async () =>{
+        const rawData = await fetch(`${process.env.REACT_APP_BACKEND_URL}/api/v1/stocks`,
+        {
+          method: "GET",
+          credentials: 'include',
```

Comment:
I have seen this very often now, what is the need of including the credentials?

Reply:
On the backend, the `api/v1/stocks` route is protected (which means a user has to supply a valid cookie that is attached to the browser). I think the `credentials` tell `fetch` to send along the cookies to the backend with the request, and then the backend analyzes the cookie to decide if the user is logged in or not.

Reply:
Hmm, I see.

Thanks for the explanation Jessie.